### PR TITLE
#419: create refer-to-mod

### PR DESCRIPTION
**Pull request by levesswe**
This is a new script regarding issue #419.  While it doesn't do all that was requested in the issue, it is a good starting point.  Additional functionality can be produced with future enhancements.  The mod worker dropdown at this time is hard-coded with Anoka County info - we may want to discuss whether that should be a global variable or some other device.

### CODE REVIEW

File: actions/refer-to-mod.vbs [hunk 1]
```diff
@@ -0,0 +1,886 @@
...
+'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("01/26/2017", "Initial version.", "Wendy LeVesseur, Anoka County")
+
+
+'Actually displays the changelog. This function uses a text file located in the My Documents folder. It stores the name of the script file and a description of the most recent viewed change.
+changelog_display
+'END CHANGELOG BLOCK =======================================================================================================
+
+'VARIABLES THAT NEED DECLARING----------------------------------------------------------------------------------------------------
+checked = 1
```
Comment (theVKC) [inline actions/refer-to-mod.vbs]: These do not require declaring when you use FuncLib.

File: actions/refer-to-mod.vbs [hunk 2]
```diff
@@ -0,0 +1,886 @@
...
+  Text 5, 40, 55, 10, "2. Press ENTER"
+  Text 5, 55, 140, 10, "3.  THEN click OK for the script to continue"
+EndDialog
+
+'THE SCRIPT----------------------------------------------------------------------------------------------------
+ 
+'Connects to BlueZone
+EMConnect ""
+
+continue = msgbox ("Run this script after you have entered the REAM for initiating an agency review.  This script can help send out documents " &_
```
Comment (theVKC) [inline actions/refer-to-mod.vbs]: Can you please use VBScript's built-in variable constants for this? See: vbYes, vbNo, vbCancel, etc.

File: actions/refer-to-mod.vbs [hunk 3]
```diff
@@ -0,0 +1,886 @@
...
+LOOP UNTIL err_msg = ""
+
+CALL check_for_PRISM(False)
+
+'The command below is necessary in order to utilize the doc_info class to efficiently obtain case data.  We are creating an object called "info" that is a member of the "doc_info" class.
+set info = new doc_info
+
+'Creating the Word application object (if any of the Word options are selected), and making it visible 
+If _
+	child_care_verification_check = checked or _
```
Comment (theVKC) [inline actions/refer-to-mod.vbs]: Nice syntax here! Really user friendly.

**Comment by levesswe** [comment]
@theVKC, the requested changes are made, as well as some bug fixes and enhancements.

**Pull request merged**